apps/api/src/app/layouts/use-cases/create-layout-change/create-layout-change.use-case.ts

```diff
@@ -0,0 +1,33 @@
+import { LayoutEntity, LayoutRepository } from '@novu/dal';
+import { ChangeEntityTypeEnum } from '@novu/shared';
+import { Injectable } from '@nestjs/common';
+
+import { CreateLayoutChangeCommand } from './create-layout-change.command';
+
+import { LayoutDto } from '../../dtos/layout.dto';
+import { ChannelTypeEnum, IEmailBlock } from '../../types';
+import { CreateChange, CreateChangeCommand } from '../../../change/usecases';
+
+@Injectable()
+export class CreateLayoutChangeUseCase {
```

Comment:
Dedicate use case for layout so it can be used in the needed use cases of the layout feature.